.github/workflows/pypi.yml

```
@@ -0,0 +1,28 @@
+name: Publish PyPI Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repo
+      uses: actions/checkout@v2.4.0
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with: 
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        rm -rf dist build *.egg-info
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+      run: |
+        python setup.py sdist bdist_wheel
+        sudo python -m twine upload dist/*  
```

Can we add the `--non-interactive` parameter ?

```suggestion
        sudo python -m twine upload --non-interactive dist/*  
```

---

I think it can be added. I just referred the existing publish.sh for this script.